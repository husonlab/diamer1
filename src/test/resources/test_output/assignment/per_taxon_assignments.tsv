node id	rank	label	kmers in database	kmer count	kmer count (accumulated)	OVO (0.20) read count	OVO (0.20) read count cumulative	OVO (0.20) read count (norm. kmers)	OVO (0.20) read count (norm. kmers) cumulative	OVO (0.50) read count	OVO (0.50) read count cumulative	OVO (0.50) read count (norm. kmers)	OVO (0.50) read count (norm. kmers) cumulative	OVO (0.60) read count	OVO (0.60) read count cumulative	OVO (0.60) read count (norm. kmers)	OVO (0.60) read count (norm. kmers) cumulative	OVO (0.80) read count	OVO (0.80) read count cumulative	OVO (0.80) read count (norm. kmers)	OVO (0.80) read count (norm. kmers) cumulative	OVO (0.90) read count	OVO (0.90) read count cumulative	OVO (0.90) read count (norm. kmers)	OVO (0.90) read count (norm. kmers) cumulative	OVO (1.00) read count	OVO (1.00) read count cumulative	OVO (1.00) read count (norm. kmers)	OVO (1.00) read count (norm. kmers) cumulative	OVA (0.70) read count	OVA (0.70) read count cumulative	OVA (0.70) read count (norm. kmers)	OVA (0.70) read count (norm. kmers) cumulative	normalized kmer count
0	no rank	root	2	1	103	1	4	1	4	1	4	1	4	1	4	1	4	1	4	1	4	1	4	1	4	1	4	1	4	1	4	1	4	0.5
1	1 from root	rank1	167	22	23	1	2	1	2	1	2	1	2	1	2	1	2	1	2	1	2	1	2	1	2	1	2	1	2	1	2	1	2	0.13173652694610777
2	1 from root	rank2	271	79	79	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	0.2915129151291513
4	2 from root	organism2	449	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0
5	2 from root	organism3	328	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	0.003048780487804878
3	2 from root	organism1	92	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0
