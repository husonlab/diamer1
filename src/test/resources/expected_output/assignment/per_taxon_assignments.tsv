node id	rank	label	kmers in database	kmer count	kmer count (accumulated)	OVO (0.20) read count	OVO (0.20) read count (norm. kmers)	OVO (0.20) cumulative read count	OVO (0.20) cumulative read count (norm. kmers)	OVO (0.50) read count	OVO (0.50) read count (norm. kmers)	OVO (0.50) cumulative read count	OVO (0.50) cumulative read count (norm. kmers)	OVO (0.60) read count	OVO (0.60) read count (norm. kmers)	OVO (0.60) cumulative read count	OVO (0.60) cumulative read count (norm. kmers)	OVO (0.80) read count	OVO (0.80) read count (norm. kmers)	OVO (0.80) cumulative read count	OVO (0.80) cumulative read count (norm. kmers)	OVO (0.90) read count	OVO (0.90) read count (norm. kmers)	OVO (0.90) cumulative read count	OVO (0.90) cumulative read count (norm. kmers)	OVO (1.00) read count	OVO (1.00) read count (norm. kmers)	OVO (1.00) cumulative read count	OVO (1.00) cumulative read count (norm. kmers)	OVA (0.70) read count	OVA (0.70) read count (norm. kmers)	OVA (0.70) cumulative read count	OVA (0.70) cumulative read count (norm. kmers)	normalized kmer count
0	no rank	root	2	2	103	2	2	4	4	2	2	4	4	2	2	4	4	2	2	4	4	2	2	4	4	2	2	4	4	2	2	4	4	1.0
1	1 from root	rank1	167	21	22	0	0	1	1	0	0	1	1	0	0	1	1	0	0	1	1	0	0	1	1	0	0	1	1	0	0	1	1	0.12574850299401197
2	1 from root	rank2	284	79	79	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	0.27816901408450706
4	2 from root	organism2	449	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0
5	2 from root	organism3	328	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	0.003048780487804878
3	2 from root	organism1	92	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0.0
